app/src/main/java/fr/free/nrw/commons/repository/UploadRepository.java

```diff
@@ -300,4 +300,8 @@ public Place checkNearbyPlaces(final double decLatitude, final double decLongitu
     public void useSimilarPictureCoordinates(ImageCoordinates imageCoordinates, int uploadItemIndex) {
         uploadModel.useSimilarPictureCoordinates(imageCoordinates, uploadItemIndex);
     }
+
+    public boolean isWMLSupportedForThisPlace() {
+        return uploadModel.getItems().get(0).isWLMUpload();
```

Comment:
As the place is same for all, checking this at least for one item will do the job